utils/create-component/templates/css/packageJson.js

```diff
@@ -0,0 +1,34 @@
+module.exports = (name, description) => `
+{
+	"name": "@workday/canvas-kit-css-${name}",
```

Comment:
Need to change these to spaces.

Reply:
Also need to add `canvas-kit-<labs>-css` here for unstable components